### PR TITLE
Add CA certs to all-in-one image

### PR DESCRIPTION
**Pull request by chandresh-pancholi**
…sticsearch backend. (#1545)

Signed-off-by: chandresh-pancholi <chandreshpancholi007@gmail.com>

## Which problem is this PR solving?
- all-in-one image misses ca-certs - can not connect to cloud-based elasticsearch backend 

## Short description of the changes
- NA

### CODE REVIEW

File: cmd/all-in-one/Dockerfile [hunk 1]
```diff
@@ -21,6 +21,7 @@ EXPOSE 14250
 # Web HTTP
 EXPOSE 16686
 
+COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
```
Comment (pavolloffay) [inline cmd/all-in-one/Dockerfile]: This is not everything, we need also these lines 

https://github.com/jaegertracing/jaeger/blob/master/cmd/collector/Dockerfile#L1-L2

Comment (chandresh-pancholi) [reply cmd/all-in-one/Dockerfile]: aha, my bad. raised the PR in a hurry. Lesson learned. 
thanks for the comment.

Comment (pavolloffay) [reply cmd/all-in-one/Dockerfile]: Also please move this `COPY` statement right after the `from scratch`, then it's ready to be merged.

**Comment by pavolloffay** [comment]
thanks @chandresh-pancholi

**Pull request merged**